libs/polysemy-wire-zoo/src/Wire/Sem/FromUTC.hs

```diff
@@ -15,16 +15,23 @@
 -- You should have received a copy of the GNU Affero General Public License along
 -- with this program. If not, see <https://www.gnu.org/licenses/>.
 
-module Spar.Sem.Now.IO
-  ( nowToIO,
-  )
-where
+-- | Time type conversion
+module Wire.Sem.FromUTC where
 
+import Data.Time (UTCTime)
 import Imports
-import Polysemy
-import SAML2.WebSSO (getNowIO)
-import Spar.Sem.Now
+import qualified SAML2.WebSSO.Types as SAML
 
-nowToIO :: Member (Embed IO) r => Sem (Now ': r) a -> Sem r a
-nowToIO = interpret $ \case
-  Get -> embed @IO getNowIO
+-- | The class is helpful in interpreters for the 'Now' effect. It makes it
+-- possible to use the interpreters for any time type so long as it implements
+-- this single-method class.
+class FromUTC a where
+  fromUTCTime :: UTCTime -> a
+
+-- An orphan instance
```

Comment:
These are not orphans!

Reply:
You're right! Silly me. I'll delete those comments.